plugin.json

```diff
@@ -7,7 +7,7 @@
     "release_notes_url": "https://github.com/mattermost/mattermost-plugin-jira/releases/tag/v3.2.0",
     "icon_path": "assets/icon.svg",
     "version": "3.2.0",
-    "min_server_version": "6.5.0",
+    "min_server_version": "7.1.0",
```

Comment:
I really dislike seeing this server version bump for this kind of PR. It's unfortunate we are excluding some customers from upgrading the plugin for other features/fixes because of this change.

It seems unlikely, but are we able to do some type assertion or something that makes it so we conditionally use something if it's available?

Reply:
7.1 is not longer a maintained ESR version. (https://forum.mattermost.com/t/support-for-extended-support-release-7-1-has-ended/15926)

Given that, I would suggest that we update `min_server_version` and also compile the plugin against that version. What do you think?

Reply:
Updated in 820ab63c41973a138bb56d8d8f883962202a1d86

Reply:
Updated in 820ab63c41973a138bb56d8d8f883962202a1d86